capa/main.py

```diff
@@ -1133,6 +1154,8 @@ def main(argv: Optional[List[str]] = None):
     if ret is not None and ret != 0:
         return ret
 
+    sys.excepthook = last_resort_exception_handler(args, *exception_information)
```

Comment:
where does `exception_information` come from? i don't see it in scope (but im on mobile so maybe i missed it).